test/e2e/config/multikueue/manager/kustomization.yaml

```
@@ -8,5 +8,5 @@ secretGenerator:
   - name: training-operator-webhook-cert
     options:
       disableNameSuffixHash: true
-patchesStrategicMerge:
-  - patch_crds.yaml
+patches:
+  - path: patch_crds.yaml
```

```suggestion
  - path: patch_crds.yaml
     target:
       version: v1
       group: apiextensions.k8s.io
       kind: CustomResourceDefinition
       name: mpijobs.kubeflow.org
```

Could you specify more explicitly?

---

Good point! Thanks!

---

Uhm. Actually I think it will not change nothing because we have on the patch:

https://github.com/kubernetes-sigs/kueue/blob/ab42df71d5747ea445d080a512930f746b435c63/test/e2e/config/multikueue/manager/patch_crds.yaml#L1-L5

---

For the future, this explicitly target allows us to avoid accidentally patching to another resource.